frontend/admin/resources/js/api/skillOperations.graphql

```diff
@@ -0,0 +1,31 @@
+query AllSkills {
+  skills {
+    id
+    key
+    name {
+      en
+      fr
+    }
+    description {
+      en
+      fr
+    }
+    keywords
+    families {
+      id
+      key
+      name {
+        en
+        fr
+      }
+      description {
+        en
+        fr
+      }
+      skills {
+        id
+      }
+      category
+    }
```

Comment:
This seems like a lot to return.  Maybe I should stub it to just families.id and then let it be expanded as needed later?

Reply:
I agree, maybe just keep the id, key, and name. In the future, we can create a fragment and let others build off of it, if need be.

Reply:
Sounds good.  I've made that change.